ci/jenkins/test-scale.sh

```diff
@@ -0,0 +1,302 @@
+#!/usr/bin/env bash
```

Comment:
I suppose this script should be integrated with Jenkins, right? If so, ci/jenkins/jobs/macros.yaml and ci/jenkins/jobs/projects-lab.yaml should also be updated.

Reply:
Will  update `ci/jenkins/test-scale.sh` `ci/jenkins/jobs/macros.yaml` `ci/jenkins/jobs/projects-lab.yaml` together.

Reply:
> Will update `ci/jenkins/test-scale.sh` `ci/jenkins/jobs/macros.yaml` `ci/jenkins/jobs/projects-lab.yaml` together.

Got it @wenqiq , I will review it again once the updates have been made.